<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-EV9AUJL" failures="3" tests="5" name="Test" time="9.888" errors="0" timestamp="2023-10-30T00:38:35 IST">
  <testcase classname="TestClassPackage.Post_TC1" name="Executer" time="6.101">
    <failure type="java.lang.AssertionError" message="expected [2023-10-29T] but found [2023-10-30T]">
      <![CDATA[java.lang.AssertionError: expected [2023-10-29T] but found [2023-10-30T]
at TestClassPackage.Post_TC1.Validator(Post_TC1.java:72)
at TestClassPackage.Post_TC1.Executer(Post_TC1.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Executer -->
  <testcase classname="TestClassPackage.Put_TC1" name="Executer" time="1.066">
    <failure type="java.lang.AssertionError" message="expected [2023-10-30T] but found [2023-10-29T]">
      <![CDATA[java.lang.AssertionError: expected [2023-10-30T] but found [2023-10-29T]
at TestClassPackage.Put_TC1.Validator(Put_TC1.java:66)
at TestClassPackage.Put_TC1.Executer(Put_TC1.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Executer -->
  <testcase classname="TestClassPackage.Delete_TC1" name="Executer" time="0.453"/>
  <testcase classname="TestClassPackage.Patch_TC1" name="Executer" time="0.971">
    <failure type="java.lang.AssertionError" message="expected [2023-10-29T] but found [2023-10-30T]">
      <![CDATA[java.lang.AssertionError: expected [2023-10-29T] but found [2023-10-30T]
at TestClassPackage.Patch_TC1.Validator(Patch_TC1.java:67)
at TestClassPackage.Patch_TC1.Executer(Patch_TC1.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Executer -->
  <testcase classname="TestClassPackage.Get_TC1" name="Executer" time="1.04"/>
</testsuite> <!-- Test -->
